Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:8 AS builder1
+
+COPY . /app/
+WORKDIR /app/
+
+RUN apt-get update && \
+    apt-get install -y zip && \
+    apt-get clean
+
+
+RUN npm install && \
+    npm run build
+
+FROM php AS builder2
```

Comment:
What about using the `composer` image so you can avoid installing composer by yourself?

https://hub.docker.com/_/composer

Reply:
Will have a look at it, that could speed up the build phase 👍